src/platform/tgatexture.cpp

```diff
@@ -93,7 +93,7 @@ detail::TextureImpl* Load( File& F )
         Succ = F.Read( Buf, InSize );
         if( Succ )
         {
-            detail::TextureImpl::ConvertRGBtoRGBA( ( uint8_t* )( void* )Buf.c_str(), ImgSize, &impl->mData.at( 0 ) );
```

Comment:
The original problem here is, according to coverity, the way Width() and Height() is returned. Maybe that could be changed? I think ImgSize is more intuitive then Buf.size() / channel.
Just an opinion, not a recommendation.